quickwit-cluster/src/cluster.rs

```diff
@@ -159,10 +204,13 @@ impl Cluster {
         let chitchat_handle = spawn_chitchat(
             chitchat_config,
             vec![
-                (GRPC_ADDRESS_KEY.to_string(), grpc_public_addr.to_string()),
+                (GRPC_ADDRESS_KEY.to_string(), me.grpc_address.to_string()),
```

Comment:
The rename is making really hard to know if the node is going to advertise its listen or advertise gRPC address. It's critical that we advertise the... wait for it...  advertise address :)

Reply:
indeed, my bad, let's go for the advertise address.